src/Resources/Tags/Tag/GetAzureTagCommand.cs

```diff
@@ -15,50 +15,86 @@
 using System.Collections.Generic;
 using System.Management.Automation;
 using Microsoft.Azure.Commands.ResourceManager.Common.Tags;
-using Microsoft.Azure.Commands.Tags.Model;
+using Microsoft.Azure.Commands.Tags.Client;
 
 namespace Microsoft.Azure.Commands.Tags.Tag
 {
     /// <summary>
-    /// Creates a new tag with the specified values
+    /// Gets predefined Azure tags | Gets the entire set of tags on a resource or subscription.
     /// </summary>
     [Cmdlet("Get", ResourceManager.Common.AzureRMConstants.AzureRMPrefix + "Tag"), OutputType(typeof(PSTag))]
     public class GetAzureTagCommand : TagBaseCmdlet
     {
-        [Parameter(Position = 0, Mandatory = false, ValueFromPipelineByPropertyName = true, HelpMessage = "Name of the tag. If not specified, return all the tags of the subscription.")]
+        #region Parameter Set Names
+
+        private const string GetPredefinedTagSet = "GetPredefinedTagSet";
+        private const string GetByResourceIdParameterSet = "GetByResourceIdParameterSet";
+
+        #endregion
+
+        #region Input Parameter Definitions
+
+        [Parameter(Position = 0, 
+            Mandatory = false, 
+            ValueFromPipelineByPropertyName = true, 
+            ParameterSetName = GetPredefinedTagSet, 
+            HelpMessage = "Name of the tag. If not specified, return all the tags of the subscription.")]
```

Comment:
Name of the tag. If not specified, return all the predefined and used tags under the subscription.